Hymns/Display/DisplayHymnView.swift

```diff
@@ -1,8 +1,9 @@
 import SwiftUI
 import Resolver
 
-struct DetailHymnScreen: View {
-    @ObservedObject private var viewModel: DetailHymnScreenViewModel
+struct DisplayHymnView: View {
```

Comment:
I think this still needs to be observed. Once @ObservedObject is removed we no longer have access to changing the @published property of a hymn being favorited or not. Is there a particular reason for removing?

Reply:
ah, okay. I didn't think about the favorites. Okay. let's keep it. Do you want to do it? or should I?

Reply:
I already did in my most recent commit. I just tagged a commit on to your PR. Mainly to fix a nested nav issue and a few other small superficial things. Take a look. I approve your stuff.